docs/src/configs/docs/deltas/entities.js

```diff
@@ -442,6 +442,17 @@ export default [
       'All the possible values for the "period" field of the delta model. (Note: a month is 31 days)',
     values: ['6h', 'day', 'week', 'month', 'year']
   },
+  {
+    name: 'Player types',
+    description:
+      'All the possible values for the "playerType" field of the delta model.',
```

Comment:
There is no playerType in the delta model, maybe change this to ...for the "playerType" query parameter.

The same applies for the playerBuild below.

Reply:
Fixed in ae1a27c